package.json

```diff
@@ -1,35 +1,56 @@
 {
-    "name": "socket.io-client"
-  , "description": "Socket.IO client for the browser and node.js"
-  , "version": "0.9.17-overleaf-3"
-  , "main" : "./lib/io.js"
-  , "browserify": "./dist/socket.io.js"
-  , "homepage": "http://socket.io"
-  , "keywords": ["websocket", "socket", "realtime", "socket.io", "comet", "ajax"]
-  , "author": "Guillermo Rauch <guillermo@learnboost.com>"
-  , "contributors": [
-        { "name": "Guillermo Rauch", "email": "rauchg@gmail.com" }
-      , { "name": "Arnout Kazemier", "email": "info@3rd-eden.com" }
-      , { "name": "Vladimir Dronnikov", "email": "dronnikov@gmail.com" }
-      , { "name": "Einar Otto Stangvik", "email": "einaros@gmail.com" }
-    ]
-  , "repository": {
-        "type": "git"
-      , "url": "https://github.com/LearnBoost/socket.io-client.git"
+    "name": "socket.io-client",
+    "description": "Socket.IO client for the browser and node.js",
+    "version": "0.9.17-overleaf-4",
+    "main": "./lib/io.js",
+    "browserify": "./dist/socket.io.js",
+    "homepage": "http://socket.io",
+    "keywords": [
+        "websocket",
+        "socket",
+        "realtime",
+        "socket.io",
+        "comet",
+        "ajax"
+    ],
+    "author": "Guillermo Rauch <guillermo@learnboost.com>",
+    "contributors": [
+        {
+            "name": "Guillermo Rauch",
+            "email": "rauchg@gmail.com"
+        },
+        {
+            "name": "Arnout Kazemier",
+            "email": "info@3rd-eden.com"
+        },
+        {
+            "name": "Vladimir Dronnikov",
+            "email": "dronnikov@gmail.com"
+        },
+        {
+            "name": "Einar Otto Stangvik",
+            "email": "einaros@gmail.com"
+        }
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/LearnBoost/socket.io-client.git"
+    },
+    "dependencies": {
+        "active-x-obfuscator": "0.0.1",
```

Comment:
Bonus: It looks like this is a dev-dependency as well (there is only one reference in the builder script). WDYT about moving it over to the other category?

Reply:
Ah.. good one. I've moved that as well.  I've reordered the last 2 commits so the release commit is the final one to avoid confusion.